unittests/Driver/DependencyGraphTests.cpp

```diff
@@ -1,54 +1,78 @@
 #include "swift/Driver/DependencyGraph.h"
+#include "swift/Frontend/ReferenceDependencyKeys.h"
 #include "gtest/gtest.h"
 
 using namespace swift;
 using LoadResult = DependencyGraphImpl::LoadResult;
+using namespace reference_dependency_keys;
+
+static LoadResult loadFromString(DependencyGraph<uintptr_t> &dg, uintptr_t node,
+                                 StringRef key, StringRef data) {
+  return dg.loadFromString(node, key.str() + data.str());
+}
+
+static LoadResult loadFromString(DependencyGraph<uintptr_t> &dg, uintptr_t node,
+                                 StringRef key1, StringRef data1,
+                                 StringRef key2, StringRef data2) {
+  return dg.loadFromString(node,
+                           key1.str() + data1.str() + key2.str() + data2.str());
+}
+
+static LoadResult loadFromString(DependencyGraph<uintptr_t> &dg, uintptr_t node,
+                                 StringRef key1, StringRef data1,
+                                 StringRef key2, StringRef data2,
+                                 StringRef key3, StringRef data3,
+                                 StringRef key4, StringRef data4) {
+  return dg.loadFromString(node, key1.str() + data1.str() + key2.str() +
+                                     data2.str() + key3.str() + data3.str() +
+                                     key4.str() + data4.str());
```

Comment:
Thanks for moving these! You could make this even cleaner by inserting the delimiters and newlines here, rather than in the callers.

Reply:
done!